packages/ckeditor5-table/tests/tablecaption/tablecaptionediting.js

```diff
@@ -0,0 +1,223 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
+import VirtualTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/virtualtesteditor';
+import { getData as getModelData, setData as setModelData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model';
+import { Plugin } from 'ckeditor5/src/core';
```

Comment:
In tests, we should use the classic imports (not those from DLLs).

Reply:
Done.